common/scripts/base.sh

```diff
@@ -139,3 +138,25 @@ function showUsageAndExitDefault() {
   echoBold "Ex: ./deploy.sh"
   exit 1
 }
+
+function deploy_common_service()
+{
+  if ! bash ${mesos_artifacts_home}/common/${1}/deploy.sh; then
+    echoError "Non-zero exit code returned when deploying ${1}"
+    exit 1
+  fi
+}
+
+function deploy_wso2_service()
```

Comment:
@ravihansa3000 Since this function is used to deploy wso2 products and the product specific dbs, named the function as 'deploy_wso2_service'. Also we already have a function called 'deploy_common_service'. I think this name is bit clear.